libs/blocks/mas/mas.js

```diff
@@ -0,0 +1,32 @@
+import { loadScript, getConfig, createTag } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+await loadScript(`${base}/deps/tacocat-index.js`);
+
+export default async function init(el) {
+  const offerDiv = el.querySelector(':scope > div:first-child > div');
```

Comment:
There is another pattern to display - a strike through price followed by a promo price.

Reply:
Thanks. This isn't meant to be comprehensive by any means. I'll see if I can add this easily.